sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/CaseSteps.java

```diff
@@ -17,17 +17,116 @@
  */
 package org.sormas.e2etests.steps.api;
 
+import static com.google.common.truth.Truth.assertThat;
+
 import cucumber.api.java8.En;
 import javax.inject.Inject;
+import org.sormas.e2etests.enums.APITestData.ErrorMessages;
 import org.sormas.e2etests.helpers.api.CaseHelper;
+import org.sormas.e2etests.helpers.api.CommunityHelper;
+import org.sormas.e2etests.helpers.api.CountryHelper;
+import org.sormas.e2etests.helpers.api.PersonsHelper;
 import org.sormas.e2etests.pojo.api.Case;
 import org.sormas.e2etests.services.api.CaseApiService;
+import org.sormas.e2etests.services.api.PostCaseBodyService;
+import org.sormas.e2etests.services.api.PostPersonBodyService;
 import org.sormas.e2etests.state.ApiState;
+import org.sormas.e2etests.state.BodyResources;
 
 public class CaseSteps implements En {
 
   @Inject
-  public CaseSteps(CaseHelper caseHelper, CaseApiService caseApiService, ApiState apiState) {
+  public CaseSteps(
+      CaseHelper caseHelper,
+      PersonsHelper personHelper,
+      ApiState apiState,
+      CaseApiService caseApiService,
+      CommunityHelper communityHelper,
+      CountryHelper countryHelper,
+      BodyResources bodyResources,
+      PostCaseBodyService bodyService,
+      PostPersonBodyService personBodyService) {
+
+    Given(
+        "I create a person",
+        () -> {
+          bodyResources.setBody("[" + personBodyService.generatePostPersonBody() + "]");
+          personHelper.pushPerson("push", bodyResources.getBody());
+          System.out.println("Create person response = " + apiState.getResponse().getStatusCode());
+        });
+
+    Given(
+        "I try to enter invalid user for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyTooOld() + "]");
+        });
+
+    Given(
+        "I try to enter valid data for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyValid() + "]");
+        });
+
+    Given(
+        "I try to enter invalid disease for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyUnknownDisease() + "]");
+        });
+
+    Given(
+        "I try to enter all lower case healthFacility for a new case",
+        () -> {
+          bodyResources.setBody("[" + bodyService.generatePostCaseBodyWithInvalidFacility() + "]");
+        });
+
+    When(
+        "I create a new case",
+        () -> {
+          caseHelper.postCases("push", bodyResources.getBody());
+        });
+
+    Then(
+        "I get the error message TOO_OLD",
```

Comment:
please rename it to make it clear that is via API, and avoid using variables name in this message